wifiphisher/pywifiphisher.py

```diff
@@ -132,10 +130,35 @@ def parse_args():
         "-iNM",
         "--no-mac-randomization",
         help=("Do not change any MAC address"), action='store_true')
+    parser.add_argument(
+        "--log-file",
+        help=("Log activity to file"))
 
     return parser.parse_args()
 
 
+VERSION = "1.3GIT"
+args = parse_args()
+APs = {}  # for listing APs
+
+filename = "wifiphisher.log"
+should_roll_over = os.path.isfile(filename)
+logger = logging.getLogger("wifiphisher")
+logger.setLevel(logging.INFO)
+
+if args.log_file:
+    file_handler = logging.handlers.RotatingFileHandler(filename, backupCount=3)
```

Comment:
Just a quick review for the code. I think instead of defining the handlers in the program we can create a configuration file as the method 2 of the following link:
[file config](https://docs.python.org/2/howto/logging.html#configuring-logging

By doing so tool users can change the `logging level` by modifying the configuration file.
@blackHatMonkey how do you think?

Reply:
But I'm not sure if it will allow us to do the `RotatingFileHandler` as we need it to rotate.